### DISCUSSION TITLE
waline部署评论区无法显示在博客页面中

### DISCUSSION THREAD

**Discussion by xbshuai** [YunYouJun/valaxy#286] [Q&A] [answered]
### Describe the bug

valaxy.config.ts完全按照文档构建，可见https://github.com/xbshuai/xbshuai.github.io ，并且waline的部署url：https://www.pangpact.space/ 也是可用的，但是最终页面里没有评论区显示。是否是因为dependencies的问题引发：
<img width="240" alt="image" src="https://github.com/YunYouJun/valaxy/assets/61312464/98628e23-f516-4de1-a537-2c1717bc7c42">
如果是，那么应该如何解决呢，我尝试过更换版本也没有作用

### Reproduction

https://github.com/xbshuai/xbshuai.github.io

### System Info

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (20) x64 13th Gen Intel(R) Core(TM) i7-13700H
    Memory: 2.26 GB / 15.64 GB
  Binaries:
    Node: 20.8.1 - D:\Node_js\node.EXE
    npm: 10.1.0 - D:\Node_js\npm.CMD
    pnpm: 8.9.2 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Edge: Chromium (118.0.2088.69)
    Internet Explorer: 11.0.22621.1
```

### Used Package Manager

pnpm

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by fanghsiu** [comment, marked as answer]
你没在 site.config.ts 打开评论
```
comment: {
  enable: true
},
```

**Comment by YunYouJun** [comment]
https://github.com/YunYouJun/valaxy/tree/main/packages/valaxy-addon-waline#%E4%B8%BB%E9%A2%98%E4%BD%BF%E7%94%A8%E8%80%85

可參照此处使用，打开 `comment.enable`:

![image](https://github.com/YunYouJun/valaxy/assets/25154432/cf1075d3-899c-4737-9a58-2679663ea389)
